include/jemalloc/internal/jemalloc_internal_types.h

```diff
@@ -94,7 +94,7 @@ typedef int malloc_cpuid_t;
 #  ifdef __powerpc__
 #    define LG_QUANTUM		4
 #  endif
-#  ifdef __riscv__
+#  ifdef __riscv
```

Comment:
To make it compatible in both cases, let’s change it to:
```#if defined(__riscv) || defined(__riscv__)```

Also please update the commit message accordingly.

Reply:
Done!